src/userscores/internal/userscoresservice.cpp

```diff
@@ -43,6 +43,7 @@ mu::ValCh<std::vector<Meta> > UserScoresService::recentScoreList() const
 
 std::vector<Meta> UserScoresService::parseRecentList(const QStringList& recentScoresPathList) const
 {
+    TRACEFUNC;
```

Comment:
what is this for?

Reply:
This is a macro for the built-in profiler.
Measures the execution time of functions
And we can display the report at any time like this

```
Main thread. Top 150 by sum time (total count: 6)
Function                                                    Call time         Call count        Sum time          
UserScoresService::recentScoreList                          7.656 ms          11                84.216 ms         
UserScoresService::parseRecentList                          7.626 ms          11                83.886 ms         
UserScoresConfiguration::recentScoreList                    0.018 ms          12                0.221 ms          
UiConfiguration::setPageState                               0.032 ms          4                 0.130 ms          
NotationPaintView::load                                     0.027 ms          2                 0.055 ms          
UiConfiguration::pageState                                  0.018 ms          1                 0.018 ms   
```

We can see which functions are called too often (for example, now there is some problem with `recentScoreList`, called too often). And we can see how long the functions take to execute, in order to decide whether to optimize them or not.